_emails/philadelphia.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Philadelphia, PA
+permalink: "/philadelphia"
+name: Letter to Mayor and Council Members
+state: PA
+city: Philadelphia
+layout: email
+recipients:
+- Cindy.Bass@Phila.gov
+- Kendra.Brooks@phila.gov
+- Kenyatta.Johnson@Phila.gov
+- Helen.Gym@phila.gov
+- Jamie.Gauthier@phila.gov
+- Maria.Q.Sanchez@phila.gov
+- Derek.Green@phila.gov
+- Katherine.Gilmore.Richardson@phila.gov
+- Allan.Domb@Phila.gov
+- Cherelle.Parker@Phila.gov
+- Isaiah.Thomas@phila.gov
+- Darrell.Clarke@phila.gov
+- David.Oh@Phila.gov
+- Brian.Oneill@phila.gov
+- Mark.squilla@phila.gov
+- brian.abernathy@phila.gov
+- james.kenney@phila.gov
+- bobby@bobbyhenon.com
+subject: Philadelphia demands a People's Budget
+body: |
+ To whom it may concern: 
+
+ My name is [insert name], and I am a resident of the [district number] district. I am emailing to demand the restructuring of the Philadelphia city budget in a way that prioritizes social services for communities and drastically minimizes spending on police. 
```

Comment:
let's use all caps for the fields inside brackets.

[NAME], [DISTRICT NUMBER], etc.